server/server.py

```diff
@@ -0,0 +1,17 @@
+import http.server
+import socketserver
+
+#set the process name to "chaos_server" so we can easily kill it with "pkill chaos_server"
+def set_proc_name(newname):
+    from ctypes import cdll, byref, create_string_buffer
+    libc = cdll.LoadLibrary('libc.so.6')
```

Comment:
Why not put the lib name in `settings.py`? Or are we just assuming it's not changing because 14.04?